### PR TITLE
Correct TODO's

### PR DESCRIPTION
**Pull request by sibille**
- Quick summary of changes
Correct TODO's
- Which issue does this PR relate to?
#3698 Login failure

### CODE REVIEW

File: code/test/Templates.Test/CodeStyleEnforcementTests.cs [hunk 1]
```diff
@@ -102,6 +102,20 @@ void EnsureUwpTemplatesNotUsed(string fileExtension)
             EnsureUwpTemplatesNotUsed("*.vb");
         }
 
+        [Fact]
+        public void EnsureCodeDoesNotUseInvalidTodoCommentIdentifier()
+        {
+            void EnsureUwpTemplatesNotUsed(string fileExtension)
```
Comment (mrlacey) [inline code/test/Templates.Test/CodeStyleEnforcementTests.cs:108]: This method name is not indicative of what it does

Comment (sibille) [reply code/test/Templates.Test/CodeStyleEnforcementTests.cs:108]: How about EnsureCodeDoesNotUseSwappedTodoComment ?
I want to test that I do not accidentially swap TODO and WTS.

Comment (mrlacey) [reply code/test/Templates.Test/CodeStyleEnforcementTests.cs:108]: Grrrr, GitHub interface. My comment was against line 108. "EnsureUwpTemplatesNotUsed" is what isn't appropriate

Comment (sibille) [reply code/test/Templates.Test/CodeStyleEnforcementTests.cs]: ah, ok, now I see it, changed!

**Pull request merged**